### DISCUSSION TITLE
Unable to ignore libraries

### DISCUSSION THREAD

**Discussion by binarykitchen** [babel/babel#12205] [Q&A]
Hello there - hope anyone can help with this:

I've listed the directory called `lib` containing libraries I don't want Babel nor core-js to touch, nor to transpile anything, in the `ignore` option. Since they are already in ES5, they will work fine on IE11. These are for example jQuery, Goog and ExtjS.

But then the coffee-loader to compile our source code complains and says:

```
ERROR in ./src/modules/ext/index.coffee
Module build failed (from ./node_modules/coffee-loader/dist/cjs.js):
CoffeeScriptError: Error: Configuration contains string/RegExp pattern, but no filename was passed to Babel
    at Object.loader (/Users/michael-heuberger/code/someapp/node_modules/coffee-loader/dist/index.js:48:14)
 @ multi jquery goog ext lib[2]
```

Huh, how come? There are plenty of other files to transpile. Why does ignoring few causes no filename at all to be passed to Babel? I've also tried the `exclude` option. Same problem.

For the record, here is my Babel config:
https://gist.github.com/binarykitchen/f08cd6d452aafedcd5b4274d2fc67e86

Can you find the cause? Any ideas what this could be?

Many thanks!

**Comment by binarykitchen** [comment]
Maybe it is this bug? Not sure: https://github.com/jashkenas/coffeescript/issues/5335

**Comment by kaicataldo** [comment]
It does seem like that might be the case - I'm not seeing anything immediately pop out in the config you shared
